Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/MicrosoftGraphDeviceManagement.py

```diff
@@ -238,6 +250,22 @@ def list_managed_devices_command(client: MsGraphClient, args: dict) -> None:
     return_outputs(human_readable, entry_context, raw_response)
 
 
+def find_managed_device_command(client: MsGraphClient, args: dict) -> None:
+    device_name: str = str(args.get('device_name'))
+    list_raw_devices, raw_response = client.find_managed_device(device_name)
+    list_devices: list = [build_device_object(device) for device in list_raw_devices if device]
+    entry_context: dict = {'MSGraphDeviceManagement.Device(val.ID === obj.ID)': list_devices}
+    human_readable: str = f'Managed device {device_name} not found.'
+    if list_devices:
+        name: str = 'List managed devices'
+        if len(list_devices) == 1:
```

Comment:
Can there be more than one?

Reply:
Sure, current version of code support such case.